### PR TITLE
feat: add basic app with header and a form

### LINKED ISSUE #10: Create the basic app
An app that shows a form with material design

### PR DESCRIPTION
**Pull request by YonatanKra**
Closes #10

### CODE REVIEW

File: apps/vaccitron/src/app/app-header/app-header.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```
Comment (MaMazav) [inline apps/vaccitron/src/app/app-header/app-header.component.spec.ts:1]: Can we split tests and production code to different folders?
(You may ignore this comment if I'm just not used to the conventions of those technologies).

Comment (YonatanKra) [reply apps/vaccitron/src/app/app-header/app-header.component.spec.ts:1]: This is the convention I work with. There's no difference for me between tests and production code besides the fact that the tests are written before the production code :)

**Comment by MaMazav** [comment]
Can you edit the Readme.md file with what prereq we need (if any) and how to build, run and test the app after clone?

**Review by MaMazav — APPROVED** [review]
1. Still recommending to put a small code snippet in the beginning of the yarn.
2. See the comment about location of test files.
(I think I created the comment on this review, still not used to work with git review :) ).
Approving to unblock, consider taking my suggestions.

**Comment by YonatanKra** [comment]
> 1. Still recommending to put a small code snippet in the beginning of the yarn.
> 2. See the comment about location of test files.
>    (I think I created the comment on this review, still not used to work with git review :) ).
>    Approving to unblock, consider taking my suggestions.

@MaMazav 
I've responded to the tests issue. I'm yet to see people writing tests ;)
But that's just how I work and this is the standard.
It doesn't matter where the tests files are - it matters that people will write them (preferable before writing the code).

I've added two lines in the readme.

**Pull request merged**